src/js/stores/CosmosPackagesStore.js

```diff
@@ -355,6 +372,18 @@ class CosmosPackagesStore extends GetSetBaseStore {
     this.emit(COSMOS_REPOSITORIES_SUCCESS);
   }
 
+  processServiceDescriptionSuccess(cosmosPackage, name) {
+    this.set({ packageDetails: cosmosPackage });
+
+    this.emit(COSMOS_SERVICE_DESCRIBE_CHANGE, name);
+  }
+
+  processServiceDescriptionError(error, name) {
```

Comment:
seems like you have this method `processPackageDescriptionError `

Reply:
Same as above. The emitted event is different.